pkg/retry/retry.go

```diff
@@ -113,6 +112,11 @@ func (s *Server) handleRetry() http.Handler {
 
 			logger.Infow("retrieve deliveries from GitHub", "cursor", cursor, "size", len(deliveries))
 
+			// store the ID of the first delivery from the first list call
+			if newCheckpoint == "" {
+				newCheckpoint = strconv.FormatInt(*deliveries[0].ID, 10)
```

Comment:
Will the next page include this checkpoint or get everything AFTER it?

Reply:
the former, it will begin with the provided cursor.

Reply:
our first list call will always start without a cursor since the events will be in descending time order.